tests/pending/pos/compound.scala

```diff
@@ -7,3 +7,7 @@ abstract class Test {
   var xx: A with B { type T; val xz: T } = null;
   xx = yy;
 }
+abstract class Test2 {
```

Comment:
What's the state of this test and the next one?
It would be good do add a comment